src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/steam/GregtechMetaTileEntity_SteamMixer.java

```diff
@@ -179,6 +207,11 @@ public ITexture[] getTexture(final IGregTechTileEntity aBaseMetaTileEntity, fina
     @Override
     public IStructureDefinition<GregtechMetaTileEntity_SteamMixer> getStructureDefinition() {
         if (STRUCTURE_DEFINITION == null) {
+            simpleBlockTiers.put(Blocks.iron_block, 1);
+
+            if (EnderIO.isModLoaded()) {
+                simpleBlockTiers.put(GameRegistry.findBlock(EnderIO.ID, "blockIngotStorage"), 6);
+            } else simpleBlockTiers.put(Blocks.iron_block, 2);
```

Comment:
Please, use unificator instead of hardcoding blocks. And add next tier only if block exists.
```suggestion
            ItemStack blockDarkSteel = GT_OreDictUnificator.get(OrePrefixes.block, Materials.DarkSteel, 1L);
            if (blockDarkSteel != null) {
                simpleBlockTiers.put(blockDarkSteel);
            }
```